pkg/controllers/securitypolicy_controller.go

```diff
@@ -217,6 +227,16 @@ func (r *SecurityPolicyReconciler) setupWithManager(mgr ctrl.Manager) error {
 			controller.Options{
 				MaxConcurrentReconciles: runtime.NumCPU(),
 			}).
+		Watches(
```

Comment:
It's better to add a few comments to explain why we add new watches here for namedport. we had put it in the design doc, and it also be better to put a brief here so that we could understand this design idea.